app/views/feeds/_tabs.html.erb

```diff
@@ -10,7 +10,17 @@
   </div>
   <% if policy(podcast).update? %>
     <div class="fixed-bottom col-sm-12 col-md-3 col-xl-2">
-      <%= link_to "Add a Feed", new_podcast_feed_path(podcast), class: "btn btn-success d-flex sticky-md-bottom m-4 shadow" %>
+      <div class="btn-group dropup d-flex sticky-md-bottom m-4 shadow">
+        <%= link_to "Add a Feed", new_podcast_feed_path(podcast), class: "btn btn-success flex-grow-1" %>
+        <% unless @feeds.select { |f| f.type == "Feeds::AppleSubscription" }.present? %>
```

Comment:
I added a `.apple?` method to Feeds awhile back.  So now could simplify this to `@feeds.none?(&:apple?)`.